tests/queries/0_stateless/01411_from_unixtime.reference

```diff
@@ -0,0 +1,29 @@
+1970-01-01 08:02:03
```

Comment:
This test result depends on your timezone,
let's write test that it will be indenpendent.

Reply:
ok.